.dev_requirements.txt

```
@@ -0,0 +1,6 @@
+ansible
+ansible-lint
+yamllint
+molecule
+molecule-docker
+molecule-podman
```

`molecule-*` packages are deprecated. Please use `molecule-plugins`.

```suggestion
molecule-plugins
molecule-plugins[docker]
molecule-plugins[podman]
```